src/components/ticket-page/InnerTicketInfo.tsx

```diff
@@ -125,6 +128,7 @@ const InnerTicketInfo = (props: InnerTicketInfoProps) => {
       'ticket-left',
       'ticket-marked-as-priority',
       'ticket-description-changed',
+      'ticket-location-description-changed',

```

Comment:
We should add this to the if statement on line 149 
`if (message === 'ticket-description-changed' || message  === 'ticket-location-description-changed') {...}` so that way  the notification is correct